docs/notebooks/integrations.rst

```diff
@@ -1,279 +1,238 @@
+    .. code-block:: Bash
 
-Run the installation script:
+        sudo apt-get update
+        sudo apt-get install \
+            apt-transport-https \
+            ca-certificates \
+            curl \
+            gnupg-agent \
+            software-properties-common
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+        sudo apt-key fingerprint 0EBFCD88
+        sudo add-apt-repository \
+        "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) \
+        stable"
+        sudo apt-get update
+        sudo apt-get install docker-ce docker-ce-cli containerd.io
 
-.. code-block:: Bash
+    c. Run the installation script:
 
-    source install_docker.sh
+    .. code-block:: Bash
 
-**Define your secrets**
+        source install_docker.sh
 
-| Go to Neptune web app, get your ``NEPTUNE_API_TOKEN`` and copy it. Then, create a password for your JupyterLab server.
-| Set those two secrets to your environment variables ``NEPTUNE_API_TOKEN`` and ``JUPYTERLAB_PASSWORD``, like below:
+5. Define your secrets.
 
-.. code-block:: Bash
+    Go to Neptune web app, get your ``NEPTUNE_API_TOKEN`` and copy it. Then, create a password for your JupyterLab server.
+    Set those two secrets to your environment variables ``NEPTUNE_API_TOKEN`` and ``JUPYTERLAB_PASSWORD``, as shown:
 
-    export NEPTUNE_API_TOKEN='your_api_token=='
-    export JUPYTERLAB_PASSWORD='difficult_password'
+        .. code-block:: Bash
 
-**Build docker image**
+            export NEPTUNE_API_TOKEN='your_api_token=='
+            export JUPYTERLAB_PASSWORD='difficult_password'
 
-Create a new file `Dockerfile`:
+6. Build the Docker image.
 
-.. code-block:: Bash
+    a. Create a new file `Dockerfile`:
 
-    nano Dockerfile
+    .. code-block:: Bash
 
-Copy insights of the following `Dockerfile` to your newly created file:
+        nano Dockerfile
 
-.. code-block:: Docker
+    b. Copy insights of the following `Dockerfile` to your newly created file:
 
-    # Use a miniconda3 as base image
-    FROM continuumio/miniconda3
+    .. code-block:: Docker
 
-    # Installation of jupyterlab and extensions
-    RUN pip install jupyterlab==0.35.6  && \
-        pip install jupyterlab-server==0.2.0  && \
-        conda install -c conda-forge nodejs
-
-    # Installation of Neptune and enabling neptune extension
-    RUN pip install neptune-client  && \
-        pip install neptune-notebooks  && \
-        jupyter labextension install neptune-notebooks
+        # Use a miniconda3 as base image
+        FROM continuumio/miniconda3
 
-    # Setting up Neptune API token as env variable
-    ARG NEPTUNE_API_TOKEN
-    ENV NEPTUNE_API_TOKEN=$NEPTUNE_API_TOKEN
+        # Installation of jupyterlab and extensions
+        RUN pip install jupyterlab==0.35.6  && \
+            pip install jupyterlab-server==0.2.0  && \
+            conda install -c conda-forge nodejs
```

Comment:
Please, let's update versions here:
```
 RUN pip install jupyterlab==1.2.6  && \
          pip install jupyterlab-server==1.0.6  && \
          conda install -c conda-forge nodejs
```

Sorry for not catching it earlier!

Reply:
NP. Fixed.